<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color: #282a36;  /* Desired background color */
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="topLogo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="BioPixelGui.qrc">:/images/images/images/BioPixelIcon_small.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="mainWorkingArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>931</width>
      <height>471</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QTabWidget::pane {
          background: #282a36;  /* Color of the tab pages */
      }
      QTabBar::tab {
          background: #282a36;  /* Color of the tabs */
          padding: 10px;
          height: 40px;
      }
      QTabBar::tab:selected {
          background: #44475a;  /* Color of the selected tab */
      }

     </string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <widget class="QWidget" name="Preferences">
     <property name="styleSheet">
      <string notr="true">
        background-color: #44475a;  /* Dark gray background */
      </string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <attribute name="icon">
      <iconset resource="BioPixelGui.qrc">
       <normaloff>:/icon/images/icon/icon_menu.png</normaloff>:/icon/images/icon/icon_menu.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
    <widget class="QWidget" name="Home">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
        background-color: #44475a;  /* Dark gray background */
      </string>
     </property>
     <attribute name="icon">
      <iconset resource="BioPixelGui.qrc">
       <normaloff>:/icon/images/icon/cil-house.png</normaloff>:/icon/images/icon/cil-house.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLineEdit" name="lineDirectory">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>591</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
        color: white;
        padding-left: 10px;
       </string>
      </property>
      <property name="font">
        <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
        </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBrowse">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string> Browse</string>
      </property>
      <property name="icon">
       <iconset resource="BioPixelGui.qrc">
        <normaloff>:/icon/images/icon/cil-folder-open.png</normaloff>:/icon/images/icon/cil-folder-open.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="labelDirectory">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>461</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Current Working Directory</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listImageFiles">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string>
          QListWidget {
              background-color: #44475a; /* Background color */
              color: white; /* Text color */
              font-family: Roboto; /* Font family */
              font-size: 12pt; /* Font size */
          }
          QListWidget::item {
              color: white; /* Text color for items */
          }
        </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkKeepTif">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
        <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Keep TIF Files</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Process">
     <property name="styleSheet">
      <string notr="true">
        background-color: #44475a;  /* Dark gray background */
      </string>
     </property>
     <attribute name="icon">
      <iconset resource="BioPixelGui.qrc">
       <normaloff>:/icon/images/icon/cil-fire.png</normaloff>:/icon/images/icon/cil-fire.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QCheckBox" name="checkDetectCells">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
        <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Detect Cells</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="topTagLine">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>461</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>BioPixel: The Future of Biomedical Image Analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="bottomCopyright">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gray;</string>
    </property>
    <property name="text">
     <string> Â© BioPixel 2024</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushExit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
       QPushButton {
           border: none;
           background: transparent;
           padding: 0px;
           color: white;
           font: bold;
       }
       QPushButton:hover {
           background: rgba(255, 255, 255, 0.1);
       }
       QPushButton:pressed {
           background: rgba(255, 255, 255, 0.2);
       }
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="BioPixelGui.qrc">
      <normaloff>:/icon/images/icon/icon_close.png</normaloff>:/icon/images/icon/icon_close.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushRun">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>550</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string> Run</string>
    </property>
    <property name="icon">
     <iconset resource="BioPixelGui.qrc">
      <normaloff>:/icon/images/icon/cil-media-play.png</normaloff>:/icon/images/icon/cil-media-play.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="BioPixelGui.qrc"/>
 </resources>
 <connections/>
</ui>
